sidebars.js

```diff
@@ -7,7 +7,26 @@ module.exports = {
       type: 'category',
       label: 'Guides',
       items: [
-        'guides/node-operator-manual',
+        {
+          type: 'category',
+          label: 'Node Operators',
+          items: [
+            'guides/node-operators/general-overview',
```

Comment:
Maybe we should avoid such nesting and put `Node operators guides` on the first level?

Reply:
It made sense when there was just one or two items, but four items in appropriate top category (Guides) is just too much in my opinion, especially when there are 16 other guides - it gets very cluttered. I would argue that we should do the opposite, for example, nest all the tooling guides (Tooling, Ejector, KAPI) in its own category.

Reply:
The kapi-guide.md and validator-ejector-guide.md seem redundant now, I suggest to remove them. 

from this commit: https://github.com/lidofinance/docs/commit/7266da5d4bf783e2c8af080d515601d8b7de39bd

Reply:
I think the point was to later on add full guides for those apps there like this one for the Ejector:
https://hackmd.io/@lido/BJvy7eWln

But I may be mistaken, any thoughts on this @avsetsin @Amuhar ?

Reply:
> I think the point was to later on add full guides for those apps there like this one for the Ejector:

Yes, that's right, these are stubs for now, we'll finish them later